query/src/main/scala/hmda/query/model/institutions/InstitutionQuery.scala

```diff
@@ -2,10 +2,30 @@ package hmda.query.model.institutions
 
 case class InstitutionQuery(
   id: String,
-  name: String,
-  cra: Boolean,
   agency: Int,
+  filingPeriod: Int,
+  activityYear: Int,
+  respondentId: String,
   institutionType: String,
-  parent: Boolean,
-  filingPeriod: Int
+  cra: Boolean,
+  emailDomain2015: String,
```

Comment:
The `InstitutionConverter` converts an `Institution` to an `InstitutionQuery`, which is what is stored in the database.  I based the database columns off of the columns in the panel CSV, so that included the year labels.  I could switch the names to something more generic, like `emailDomain1`, if that would be less confusing.